### DISCUSSION TITLE
Acesso rdmoto

### DISCUSSION THREAD

**Discussion by luis007007** [community/community#132442] [Repositories]
### Select Topic Area

Show & Tell

### Body

Você consegue construir um repositório com o s as cesso?

**Comment by luis007007** [comment]
Eu fui violado por já faz um mês e sinto que deveria ter um acesso também , ele viu que eu descobri quem quem era e apagou o repositório e eu não sei fazer

**Comment by LiteBrite82** [comment]
Hi @luis007007 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
